source/query/maxtimems.txt

```diff
@@ -14,9 +14,11 @@ interrupts the operation.
    :alt: MaxTimeMS Option
 
 The default :guilabel:`MAX TIME MS` value is 60000, or 60 seconds.
-Consider raising this value if you have a very large collection, or
-your operations are frequently timing out. You can also consider creating
-:ref:`indexes <compass-indexes>` to improve query performance.
+Consider raising this value if you have a very large collection, if 
```

Comment:
[question] is there a way to more concretely quantify a "very large" collection?

Reply:
There is no limit on the number of documents in a non-capped collection, but I think this depends on the hardware.